FullstackDevelopment/Fullstack.md

```diff
@@ -0,0 +1,395 @@
+# Fullstack Development Learning Resources
+
+Welcome to the **Fullstack Development Learning Resources** repository! This repository offers a curated list of resources that will guide you through the journey of mastering both frontend and backend technologies, as well as the skills needed to bridge the two. Whether you're just starting or advancing your fullstack development knowledge, these resources are here to help.
+
+
+## 🛣️ Fullstack Development Roadmap
+
+Here’s a visual roadmap to guide you through the fullstack development journey:
+
+![Fullstack Development Roadmap](https://github.com/Rachel-Cellulose/LearningResources/blob/fullstack-development/FullstackDevelopment/assets/fullstack-roadmap.jpg) 
+
+
+## 📚 Table of Contents
+1. [🚀 Getting Started with Fullstack Development](#-getting-started-with-fullstack-development)
```

Comment:
@Rachel-Cellulose 
Your project is good

The only problem now is if I click on the table of contents it doesn't take me to that section instead I'm directed to a new page entirely

And this page looks like a web page with just html and no CSS

You can look at the readme file to see the difference